### PR TITLE
Add Digital Image Processing Course

### PR DESCRIPTION
**Pull request by MasterSaptak009**
## What does this PR do?
Add resource(s) 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -501,6 +502,9 @@
 * [Digital Electronics](https://www.youtube.com/playlist?list=PLBlnK6fEyqRjMH3mWf6kwqiTbT798eAOm) - Neso Academy
 * [Digital Electronics for GATE](https://www.youtube.com/playlist?list=PLWPirh4EWFpHk70zwYoHu87uVsCC8E2S-) - Tutorials Point India Ltd.
 
+### Digital Image Processing
+
+* [IMAGE PROCESSING USING MATLAB](https://youtube.com/playlist?list=PLEo-jHOqGNyUWoCSD3l3V-FjX9PnHvx5n) - Dr. Rashi Agarwal
```
Comment (shrn01) [inline courses/free-courses-en.md:507]:
```suggestion
* [Image Processing using Matlab](https://youtube.com/playlist?list=PLEo-jHOqGNyUWoCSD3l3V-FjX9PnHvx5n) - Rashi Agarwal
```

Dont put the title in all caps. Also we dont allow honorifications like `Dr.`

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -501,6 +502,9 @@
...
```
Comment (shrn01) [inline courses/free-courses-en.md:508]: Add 2 spaces after each section

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -501,6 +502,9 @@
 * [Digital Electronics](https://www.youtube.com/playlist?list=PLBlnK6fEyqRjMH3mWf6kwqiTbT798eAOm) - Neso Academy
 * [Digital Electronics for GATE](https://www.youtube.com/playlist?list=PLWPirh4EWFpHk70zwYoHu87uVsCC8E2S-) - Tutorials Point India Ltd.
```
Comment (shrn01) [inline courses/free-courses-en.md:504]: Add 2 spaces after each section

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -31,6 +31,7 @@
     * [Ansible](#ansible)
     * [Chef](#chef)
 * [Digital Electronics](#digital-electronics)
+* [Digital Image processing](#digital-image-processing)
```
Comment (eshellman) [inline courses/free-courses-en.md:34]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Use MATLAB or Misc categories, or start a "Graphics Programming" category, moving the "Computer Grephics" couse from Misc.

**Comment by MasterSaptak009** [comment]
please check now 
I think I have solved the issues as you said ☺

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**